distribution/changelog.txt

```diff
@@ -187,7 +187,7 @@
 - Feature: Finance window accessible from toolbar (enabled in settings).
 - Feature: Research and development / research funding now accessible as a stand alone window without the requirement of the finances window.
 - Feature: Extra viewport windows.
-- Feature: Park window viewport is resizable.
+- Feature: Park window viewport is resizeable.
```

Comment:
resizable is prefential spelling. (resizeable isn't wrong per se, but less common.)